python/tests/test_ld_matrix.py

```diff
@@ -1034,7 +1034,7 @@ def pi2_unbiased(
         w_Ab = state[1, k]
         w_aB = state[2, k]
         w_ab = n - (w_AB + w_Ab + w_aB)
-        with suppress_division_by_zero_warning():
+        with np.errstate(over="ignore", divide="ignore", invalid="ignore"):
```

Comment:
Wouldn't it be better to update the original function here rather than duplicating?

Reply:
Fixed in a8cb721b